src/main/kotlin/net/leanix/vsm/sbomBooster/service/OrtService.kt

```diff
@@ -34,7 +34,7 @@ class OrtService(
     fun downloadProject(projectUrl: String, username: String, gitToken: String): String {
         val downloadFolder = "${projectUrl.substringAfterLast("/")}_${List(10) { charPool.random() }.joinToString("")}"
         val args = mutableListOf(
-            "docker",
+            "sudo", "docker",
```

Comment:
won't this ask for password?

Reply:
Normally not because the user should already have the needed permissions. I am out of ideas for this and this was the only option.